cocos/base/ccConfig.h

```diff
@@ -162,7 +162,11 @@ THE SOFTWARE.
  * If a device doesn't support VAO though it claims to support should add exceptions list here.
  */
 #ifndef CC_TEXTURE_ATLAS_USE_VAO
-#define CC_TEXTURE_ATLAS_USE_VAO 1
+#if (CC_TARGET_PLATFORM == CC_PLATFORM_OHOS)
+    #define CC_TEXTURE_ATLAS_USE_VAO 0 // TBD ohos doesn't support VAO
```

Comment:
VAO is a standard of OpenGLES 3.x. If ohos supports OpenGLES 3.x, it should be able to support VAO.

Reply:
yes, ohos supports GLES3, but not support GLES2 yet. if we set 1 here, it will allow to execute the follow code. Therefore, we turned off support for VAO and blocked the following code through macros
![image](https://github.com/cocos2d/cocos2d-x/assets/57482234/b4561561-8beb-4d16-b881-1a4ad98979fa)

Reply:
So you mean that there is not a `glMapBufferOES` extension in GLES2 implementation on OH platform?

Reply:
<img width="963" alt="企业微信截图_02435cc0-c72c-4384-bca4-e2727e3d18df" src="https://github.com/cocos2d/cocos2d-x/assets/493372/d8beb1df-1393-478e-ab82-6efae74ec59d">

Reply:
yes, no gles2 on ohos now. this header file is pre-added.
in the cmakelist, no link to GLES2 now